### ISSUE TITLE
Tidying input checking and r5r option setting

### ISSUE THREAD

**Issue by dhersz** [ipeaGIT/r5r#46]
Each one of us were writing error and warning messages in our own way. I started using `checkmate::` where possible and where not possible I'm trying to emulate their messages as closely as possible.

I also created some more `utils` function to standardise the way we set r5r options (sometimes it would happen in the body of the main functions, as with number of threads and speeds, sometimes with the aid of utils functions, such as with verbose).

I have already made big progress with this in `detailed_itineraries`. I'll later do it with `travel_time_matrix` if you don't make any objections (feel free to do so, though).

**Comment by rafapereirabr** [comment]
I agree we should start organizing the code. I agree the `checkmate::assert_` functions are very helpful. I've used it to create a new support function `set_nThreads()`. This one has now been implemented o both routing functions.

From what I see, there is at the moment one input that is being treated/tested slightly differently between the two routing functions: 
- origins/destinations . We currently have two overlapping functions `test_points_input` and `assert_points_input`. The `assert_points_input` functions looks much more robust so I believe we should stick to it.

**Comment by dhersz** [comment]
Cool! I created the assert function today to encompass both sf_to_df and
test_point that existed previously, but since I was focusing in tidying
detailed_itinerafies first I didn't want to make any changes that would
break travel_time_matrix.

I can make such changes tomorrow.

Em qui, 20 de ago de 2020 22:37, Rafael H M Pereira <
notifications@github.com> escreveu:
